x-pack/plugins/apm/kibana.json

```diff
@@ -1,13 +1,23 @@
 {
   "id": "apm",
-  "server": true,
   "version": "8.0.0",
   "kibanaVersion": "kibana",
-  "configPath": [
-    "xpack",
-    "apm"
+  "requiredPlugins": [
+    "features",
+    "apm_oss",
+    "data",
+    "home",
+    "licensing",
+    "triggers_actions_ui"
```

Comment:
this should probably be in optional, with the other plugins necessary for alerting (`alerting` and `actions`).

Reply:
`actionTypeRegistry` and `alertTypeRegistry` are required for `<AlertsContextProvider>`, so it will have to be required unless we do some kind of conditional rendering of context, but i don't think that will be good for downstream consumers of that context.